ui/src/artist/ArtistShow.js

```diff
@@ -23,26 +25,13 @@ const useStyles = makeStyles(
       '& .MuiTypography-h5': {
```

Comment:
Please don't use this type of selector. always prefer creating a new entry here in the makeStyles and set the component `className` property to this (like in the `bioBlock` below)